test/Microsoft.AspNetCore.Routing.Tests/TestObjects/TestEndpoint.cs

```diff
@@ -0,0 +1,12 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace Microsoft.AspNetCore.Routing.Tests.TestObjects
+{
+    internal class TestEndpoint : Endpoint
```

Comment:
Thoughts on concrete test objects like this vs Moq? I lean towards test objects

Reply:
I think this is fine.

Reply:
I like concrete test objects for common things 👍